packages/pluggableWidgets/accordion-web/src/Accordion.tsx

```diff
@@ -47,3 +37,38 @@ export function Accordion(props: AccordionContainerProps): ReactElement | null {
         />
     );
 }
+
+function translateGroups(groups: AccordionContainerProps["groups"]): AccordionGroups | undefined {
+    if (
+        groups.some(
+            group =>
+                group.visible.value === undefined ||
+                group.headerText.value === undefined ||
+                group.initiallyCollapsed.value === undefined ||
+                (group.collapsed && group.collapsed.value === undefined)
```

Comment:
My bad, I was actually only referring to this specific `some` callback, but this works too 😄

Reply:
Refactored it as well 😄